include/swift/Runtime/Win32Strings.h

```diff
@@ -25,7 +25,7 @@ namespace swift {
 /// @param str The string to convert.
 ///
 /// @returns The string, converted to UTF-8. The caller is responsible for
-///   freeing this string when done with it.
+///   freeing this string with @c free() when done with it.
```

Comment:
My two suggestions collided. This should now say, with `swift_freeMetadataSectionName`.

Reply:
D'oh!

Reply:
Er, no, this one shouldn't. This function is a generic wrapper around the UTF-16-to-8 converter in Win32. `swift_copyMetadataSectionName()` should have such a comment perhaps, but currently it has _no_ comment and is only used by the test suite anyway.

Reply:
Oops, I got mixed up. Carry on, then!